deploy.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+
+# From https://gist.github.com/domenic/ec8b0fc8ab45f39403dd
+
+echo "[TRACE] TRAVIS_BRANCH: $TRAVIS_BRANCH"
+echo "[TRACE] TRAVIS_PULL_REQUEST_BRANCH: $TRAVIS_PULL_REQUEST_BRANCH"
+echo "[TRACE] TRAVIS_PULL_REQUEST: $TRAVIS_PULL_REQUEST"
+
+SOURCE_BRANCH="$TRAVIS_PULL_REQUEST_BRANCH"
+TARGET_BRANCH="$TRAVIS_PULL_REQUEST-$TRAVIS_PULL_REQUEST_BRANCH"
```

Comment:
This should be:

`TARGET_BRANCH="$TRAVIS_PULL_REQUEST_BRANCH-build"`

It makes it easier to find in the list of branches, and also makes it obvious it is a build of the PR branch

Reply:
fixed